command/polybftsecrets/utils.go

```diff
@@ -0,0 +1,54 @@
+package polybftsecrets
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/0xPolygon/polygon-edge/secrets/helper"
+)
+
+// common flags for all polybft commands
+const (
+	DataPathFlag = "data-dir"
+	ConfigFlag   = "config"
+
+	DataPathFlagDesc = "the directory for the Polygon Edge data if the local FS is used"
+	ConfigFlagDesc   = "the path to the SecretsManager config file, if omitted, the local FS secrets manager is used"
+)
+
+// common errors for all polybft commands
+var (
+	ErrInvalidNum                     = fmt.Errorf("num flag value should be between 1 and %d", maxInitNum)
+	ErrInvalidConfig                  = errors.New("invalid secrets configuration")
+	ErrInvalidParams                  = errors.New("no config file or data directory passed in")
+	ErrUnsupportedType                = errors.New("unsupported secrets manager")
+	ErrSecureLocalStoreNotImplemented = errors.New(
```

Comment:
Nitpick: You could have made all errors, except `ErrInvalidParams` local (as they are used only in `polybftsecrets` package).

Reply:
There  is another task to split register command. For those purposes I'll need these errors. 

`What should we do with command/rootchain/emit/emit.go and command/rootchain/initcontracts/init_contracts.go? Those two commands are expecting hex-encoded private key to be provided via the CLI flag, but not sure if is it ok to leave it like that.`  - this should be addressed via another task.

`Also just mark DataPathFlag and ConfigFlag mutually exclusive in each command it accommodates it.`  - will do

Reply:
done: https://github.com/0xPolygon/polygon-edge/pull/1251/commits/5f2f7453acce35cc3e7352c75442459c60908a31